apis/v1alpha1/baselineadminnetworkpolicy_types.go

```diff
@@ -105,6 +105,9 @@ type BaselineAdminNetworkPolicyIngressRule struct {
 	From []AdminNetworkPolicyPeer `json:"from"`
 
 	// Ports allows for matching traffic based on port and protocols.
+	// This field is a list of ports which should be made accessible on
```

Comment:
same as above I don't think it's always "made accessible" right? It's really just that this rule matches the specified destination ports

Reply:
yeah +1 to your point, let me change this. I blatantly copied this from netpolv1 docs hehe which are clearly not clear enough :)